.github/workflows/tiiuae-pixhawk.yaml

```diff
@@ -144,7 +144,7 @@ jobs:
     name: upload builds to Artifactory
     runs-on: ubuntu-latest
     needs: [pixhawk4, saluki-v1, saluki-v2, px4fwupdater]
-    if: github.event_name == 'push'
+    if: true
```

Comment:
Just a question; does this mean that every PR build will be pushed into the list in here https://artifactory.ssrc.fi/ui/repos/tree/General/gen-public-local ? The problem with this list is already that it is next to impossible to find anything in here. The buids from PR:s should be in some totally different "dev folder" and only builds from master branch in the original list.

Maybe we should do this one on a separate PR later, and also re-organize the way how images are arranged in the artifactory at the same?